ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/IndexMappingTool.java

```diff
@@ -32,18 +32,19 @@
 
 import lombok.Getter;
 import lombok.Setter;
+import lombok.extern.log4j.Log4j2;
 
+@Log4j2
 @ToolAnnotation(IndexMappingTool.TYPE)
 public class IndexMappingTool implements Tool {
     public static final String TYPE = "IndexMappingTool";
     public static final String STRICT_FIELD = "strict";
     private static final String DEFAULT_DESCRIPTION = String
         .join(
             " ",
-            "This tool gets index mapping information from a certain index.",
-            "It takes 1 required argument named 'index' which is a comma-delimited list of one or more indices to get mapping information from, which expands wildcards.",
-            "It takes 1 optional argument named 'local' which means whether to return information from the local node only instead of the cluster manager node (Default is false).",
-            "The tool returns a list of index mappings and settings for each index.",
```

Comment:
```
private static final String DEFAULT_DESCRIPTION = String
    .join(
        " ",
        "This tool returns index mappings and settings for specified indices.",
        "Required argument: 'index' - comma-delimited list of indices (supports wildcards like 'my-index-*').",
        "Optional argument: 'local' - if true, returns info from local node only instead of cluster manager (default: false).",
        "Response format: For each index, 'mappings' contains field definitions under 'properties' (each field has a 'type'), and 'settings' contains configuration as a flattened key-value map."
    );
```

What do you think about this?